### PR TITLE
added update_or_create function in ElementService.py

### PR DESCRIPTION
**Pull request by yyzz1010**
added update_or_create function in ElementService.py

### CODE REVIEW

File: TM1py/Services/ElementService.py [hunk 1]
```diff
@@ -67,6 +67,12 @@ def exists(self, dimension_name: str, hierarchy_name: str, element_name: str, **
             element_name)
         return self._exists(url, **kwargs)
 
+    def update_or_create(self, dimension_name: str, hierarchy_name: str, element: Element, **kwargs) -> Response:
+        if self.exists(dimension_name=dimension_name, hierarchy_name=hierarchy_name, element=element.name, **kwargs):
```
Comment (MariusWirtz) [inline TM1py/Services/ElementService.py:71]: The function exists requires the argument `element_name`

TypeError: ElementService.exists() missing 1 required positional argument: 'element_name'

Comment (yyzz1010) [reply TM1py/Services/ElementService.py]: fixed exists function argument bug

**Comment by yyzz1010** [comment]
image
new element is created with the new function, and existing element data type is updated with the new function

**Comment by MariusWirtz** [comment]
Thank you @yyzz1010

**Review by MariusWirtz — APPROVED** [review]

**Pull request merged**